### PR TITLE
[Fix](bangc-ops): check kernel launch and specify header file

### PR DESCRIPTION
**Pull request by chqy99**
Thanks for your contribution and we appreciate it a lot. 

## 1. Motivation

mluops仓库中误用LOG(ERROR)，误用kernel lauch检查，部分头文件引用顺序不规范

## 2. Modification

- host端已经做了防呆处理，对应的device端删除了部分LOG(ERROR)。
- 修正kernel launch检查，正确使用KERNEL_CHECK与CHECK_RETURN，修改kernel入口函数返回类型。
- 规范头文件引用。

#### 3.1.1 Accuracy Acceptance Standard

For static threshold standard details, see: [MLU-OPS Accuracy Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Accuracy-Acceptance-Standard.md).

- [ ] diff1: diff1 <= 3e-3
- [ ] diff2: diff2 <= 3e-3

#### 3.1.2 Operator Scheme checklist

none

#### 3.1.3 New Feature Test

none

#### 3.1.4 Parameter Check

none

### 3.2 Accuracy Test

For the cases used in the New Feature Test section, the features and the number of cases are recorded here. When multiple operations are tested, multiple tables are needed to include details of these operations.

Operation:

|Test Point           | Description                      | Quantity |  Comment |
|----------           |----------------------------------|----------|  --------|
|Data type test       |half/float/int8                   |          |          |
|Mult-tensor test     |Supports 1-8 dims                 |          |          |
|Layout test          |Supports NCHW/NHWC                |          |          |
|Zero element test    |Whether to support this test      |          |          |
|Stability test       |--gtest_repeat=NUM<br>--thread=NUM|          |          |
|mlu-only mode test   |--mlu-only,see [MLU-OPS Performance Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Performance-Acceptance-Standard.md)|          |          |
|Mult-platform test   |MLU370/MLU590                     |          |          |
|Nan/INF test         |Whether to support this test      |          |          |
|Memory leak check    |Test result                       |          |          |
|Code coverage check  |Test result                       |          |          |

MLU370流水通过

MLU590流水通过

### 3.3 Performance Test

none

### 3.4 Summary Analysis

MLU370、590流水测试通过

### CODE REVIEW

File: bangc-ops/kernels/ball_query/ball_query_union1.mlu [hunk 1]
```diff
@@ -328,6 +331,7 @@ void MLUOP_WIN_API KernelBallQuery(cnrtDim3_t k_dim, cnrtFunctionType_t k_type,
                                    const void *new_xyz, const void *xyz,
                                    int32_t *idx) {
   switch (d_type) {
+    // host端cpp文件中已做float和half型防呆
```
Comment (PetrelYy) [inline bangc-ops/kernels/ball_query/ball_query_union1.mlu:334]: 注释改为英文

File: bangc-ops/kernels/moe_dispatch_backward_gate/moe_dispatch_backward_gate_union1.mlu [hunk 2]
```diff
@@ -364,22 +364,14 @@ void MLUOP_WIN_API KernelMoeDispatchBwdGate1(
     const int capacity, const int hidden, const int num_experts,
     void *workspace, void *grad_gates) {
   switch (d_type) {
+    // host端cpp文件中只支持float型
```
Comment (PetrelYy) [inline bangc-ops/kernels/moe_dispatch_backward_gate/moe_dispatch_backward_gate_union1.mlu:367]: 按 PR784 & PR789 修改 launch kenrel 方式

File: bangc-ops/kernels/ball_query/ball_query_union1.mlu (hunk 1 above)
Comment (chqy99) [reply bangc-ops/kernels/ball_query/ball_query_union1.mlu]: 已修改

File: bangc-ops/kernels/moe_dispatch_backward_gate/moe_dispatch_backward_gate_union1.mlu (hunk 2 above)
Comment (chqy99) [reply bangc-ops/kernels/moe_dispatch_backward_gate/moe_dispatch_backward_gate_union1.mlu]: 重新改写

File: bangc-ops/kernels/carafe/carafe_block.mlu [hunk 3]
```diff
@@ -453,6 +453,8 @@ void MLUOP_WIN_API KernelCarafeForward(
     int grid_dimG,     // nb. of group blocks
     int grid_dimC) {   // nb. of channel blocks
   switch (d_type) {
+    /* Only float and half data types are supported
+       in host-side CPP file fool-proof processing. */
     case MLUOP_DTYPE_FLOAT: {
       MLUKernelCarafeForward<<<k_dim, k_type, queue>>>(
```
Comment (DanieeelLiu) [inline bangc-ops/kernels/carafe/carafe_block.mlu:459]: KERNEL_CHECK

Comment (chqy99) [reply bangc-ops/kernels/carafe/carafe_block.mlu]: 已修改

**Review by DanieeelLiu — APPROVED** [review]

File: bangc-ops/kernels/active_rotated_filter/active_rotated_filter.cpp [hunk 4]
```diff
@@ -175,9 +175,10 @@ mluOpStatus_t MLUOP_WIN_API mluOpActiveRotatedFilterForward(
   VLOG(5) << "Launch Kernel KernelActiveRotatedFilterForward<<<Union"
           << k_type / CORE_DIM << ", " << k_dims.x << ", " << k_dims.y << ", "
           << k_dims.z << ">>>.";
-  KERNEL_CHECK(KernelActiveRotatedFilterForward(
-      k_dims, k_type, handle->queue, input_dtype, output_planes, input_planes,
-      orientations, kH, kW, rotations, input, indices, workspace, output));
+  CHECK_RETURN("[mluOpActiveRotatedFilterForward]",
```
Comment (duzekunKTH) [inline bangc-ops/kernels/active_rotated_filter/active_rotated_filter.cpp:178]: 这里用check return，当函数有问题的时候，没有执行  GEN_CASE_END() 就返回报错了，这是否会引起出错时 gencase 模块出错？

**Review by PetrelYy — APPROVED** [review]

**Review by duzekunKTH — APPROVED** [review]

**Pull request merged**